compose.yml

```diff
@@ -0,0 +1,15 @@
+services:
+  app:
+    build: .
+    command: ["python", "main.py"]
+  dev:
+    build: .
+    command: ["bash"]
+    tty: true
+    volumes:
+      # Maps the current directory to /home/llcsim in the container
+      # NOTE: If the app has also been run locally, directories like
+      # __pycache__ will also map to the container with this volume.
+      # It may be necessary to remove those directories if conflicts
+      # or odd behavior arise.
```

Comment:
I think this will mostly be fine for `__pycache__` directories, but it is a little awkward to map the directory with `.gitignore`, the README, etc. I'm wondering if we want to move all the application code into the `src` directory and just map that as a volume. I think we could also put `rwims.din` and `trace.txt` in a `data` directory in the `src` directory. Open to other thoughts on directory structure though.